source/manual/alerts/unicorn-herder.html.md

```diff
@@ -24,7 +24,8 @@ Configure and activate your fabric environment as described in the
 Then run the `vm.bodge_unicorn` task for the affected machine and application:
 
 ```bash
-fab $environment -H whitehall-backend-1.backend vm.bodge_unicorn:whitehall
+# Replace the machine_ip with that of the machine that is showing the error in Icinga
+fab $environment -H $machine_ip vm.bodge_unicorn:whitehall
```

Comment:
I think you'd also have to replace `whitehall` with the relevant app name

Reply:
Good catch.